terraform-infra-approvals/darts-shared-infrastructure.json

```
@@ -30,7 +30,8 @@
     {"type": "azurerm_service_plan"},
     {"type": "azurerm_storage_account_network_rules"},
     {"type": "azurerm_logic_app_standard"},
-    {"type": "azurerm_private_endpoint"}
+    {"type": "azurerm_private_endpoint"},
+    {"type": "postgresql_extension"}
```

```suggestion
    {"type": "azurerm_private_endpoint"}
```

This isn't needed, you can enable extensions on flexible servers via the module.

```terraform
pgsql_server_configuration = [
    {
      name  = "azure.extensions"
      value = "DBLINK,PG_STAT_STATEMENTS"
    },
    {
      name  = "backslash_quote"
      value = "on"
    },
    #{
    #  name  = "TimeZone"
    #  value = "Europe/London"
    #},
    {
      name  = "pg_qs.query_capture_mode"
      value = "ALL"
    },
    {
      name  = "pgms_wait_sampling.query_capture_mode"
      value = "ALL"
    }
  ]
```